src/main/java/roomescape/controller/dto/response/MyReservationResponse.java

```diff
@@ -0,0 +1,25 @@
+package roomescape.controller.dto.response;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import roomescape.entity.Reservation;
+
+public record MyReservationResponse(
+    Long reservationId,
+    String theme,
+    @JsonFormat(pattern = "yyyy-MM-dd") LocalDate date,
+    @JsonFormat(pattern = "HH:mm") LocalTime time,
+    String status
+) {
+
+    public static MyReservationResponse from(Reservation reservation) {
+        return new MyReservationResponse(
+            reservation.getId(),
+            reservation.getTheme().getName(),
+            reservation.getDate(),
+            reservation.getTime().getStartAt(),
```

Comment:
```suggestion
            reservation.getStartAt(),
```
같이 `getStartAt`를 만들어서 반환도 가능할 것 같아요~

Reply:
만들어서 반화해보았습니다 !!